apps/ledger-live-mobile/src/components/RootNavigator/AccountsNavigator.tsx

```diff
@@ -75,6 +79,24 @@ export default function AccountsNavigator() {
           headerShown: false,
         }}
       />
+      {accountListUIFF?.enabled && (
+        <>
+          <Stack.Screen
+            name={ScreenName.AccountsList}
+            component={AccountsList}
+            options={{
+              headerTitle: "",
+            }}
+          />
+          <Stack.Screen
```

Comment:
you already have a navigator for this screen. you don't need to add it here also.